doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+Then you can see the result of boot order. Also you can check the boot order with below command:
+# efibootmgr -v            	
+or setting boot order with this command to make sure the acrn.efi is the first boot option:
+# efibootmgr -o x, x       	
+Note: by default, the “ACRN Hypervisor” you just added should be the first one to boot. But please make sure it is, or select it 
+manually after press “F10” when system booting up.   
+
+Step #10, Deploy the SOS kernel
+# sudo cp ~/kernel-build/usr/lib/kernel/org.clearlinux.pk414-sos.4.14.41-39 /boot/acrn/
+
+Step #11, Customize the boot of the grub entry 
+Use grub custom file (/etc/grub.d/40_custom) to create a new grub entry
+Add the following to the end of grub file
+menuentry 'ACRN ubuntu SOS' {
+        recordfail
+        load_video
+        insmod gzio
+        insmod part_gpt
+        insmod ext4
+        linux /boot/acrn/org.clearlinux.pk414-sos.4.14.41-39 pci_devices_ignore=(0:18:1) maxcpus=1 console=tty0 console=ttyS0 
+        i915.nuclear_pageflip=1 root=PARTUUID=<ID of rootfs partition> rw rootwait ignore_loglevel no_timer_check consoleblank=0 
+        i915.tsd_init=7 i915.tsd_delay=2000 i915.avail_planes_per_pipe=0x00000F i915.domain_plane_owners=0x011111110000 
+        i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2 
+        i915.enable_gvt=1 hvlog=2M@0x1FE00000
+}
+Note: here you have to use PARTUUID, UUID doesn’t work.  
+
+Step #12,  Update grub
+# sudo update-grub
+At this moment, you need to modify /boot/grub/grub.cfg file manually to enable the timeout so that the system has opportunity to show 
+you grub menu to select. Or you need to set the “ACRN ubuntu SOS” as the default boot option. Without enabling  timeout, the system 
+will not show grub menu for people to select and boot into incorrect kernel.
+
+Enable timeout:
+ if [ x$feature_timeout_style = xy ] ; then
+ -  set timeout_style=hidden
+ + #set timeout_style=hidden
+ -  set timeout=0
+ + set timeout = 10 
+ # Fallback hidden-timeout code in case the timeout_style feature is
+  # unavailable.
+
+Step #13, Check the boot process 
+Reboot system to see if the acrn.efi is loaded successfully, and if the Menu of Grub being shown to select “ACRN ubuntu SOS”.  
+after boot successfully, the system will start to Ubuntu Desktop successfully.  to check if the system is running on virtualization 
+evn and the ACNR hypervisor is running. 
+For example:
+   
+NUC6CAYH:~$ dmesg | grep ACRN
+[    0.000000] Hypervisor detected: ACRN
+[    0.862942] ACRN HVLog: acrn_hvlog_init
+
+This output means the boot flow is correct and the SOS system with ACRN is running well.
+ 
+
+Step #14, Deploy Clear Linux as the UOS 
+Download clear linux UOS from download.clearlinux.org
+#cd /root
```

Comment:
This will fail if you're not root so we should add a preceding `sudo -s` instruction and get rid of all the `sudo` below, until the end of the doc (since we are now `root`).